config/optional/core.entity_view_display.paragraph.localgov_featured_subsite.default.yml

```diff
@@ -0,0 +1,55 @@
+uuid: 5e3182be-026a-4bb1-9523-afcff0640504
```

Comment:
We need to remove this UUID line from the code.

Reply:
Thanks @markconroy the localgov_paragraphs repo does not contain a config/options folder, only a config/install. Should I create a config/options folder in the localgov_paragraphs repo to put this file in?